store/store.go

```diff
@@ -1104,6 +1147,42 @@ func (s *Store) Sections(user *auth.UserState) ([]string, error) {
 	return sectionNames, nil
 }
 
+// WriteCommandsCatalogs queries the "commands" endpoint and writes
+// the command names into the given io.Writer [and at some point the
+// CNF catalog to another io.Writer; adding now to minimise interface
+// churn].
+func (s *Store) WriteCommandsCatalogs(names io.Writer, _ io.Writer) error {
```

Comment:
The function name and docs would benefit from some cleaning up after the repurposing of this interface. _WriteCatalogs_ alone sounds pretty reasonable as a name, and we can also drop the second argument while it's missing on _both ends_ of the function call.

Reply:
ok, fair